### PR TITLE
Added additional account cmdlet docs

### PR DESCRIPTION
**Pull request by hnrkndrssn**
Added docs and examples for:
- Token accounts
- AWS accounts
- Username/password accounts

### CODE REVIEW

File: docs/infrastructure/deployment-targets/dynamic-infrastructure/aws-accounts.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+|-------------------------------|------------------------------------------------------------------------------------------------------------|
+| `-name`                       | Name for the AWS account                                                                                   |
+| `-secretKey`                  | The AWS secret key to use when authenticating against Amazon Web Services.                                 |
+| `-accessKey`                  | The AWS access key to use when authenticating against Amazon Web Services.                                 |
+| `-updateIfExisting`           | Will update an existing account with the same name, create if it doesn't exist                             |
+
+Example:
+```powershell
+New-OctopusAwsAccount -name "My AWS Account" `
+                      -secretKey "7U4MhdfjgcAk9niwPgXD81pTYY+fIvVsN3m" `
```
Comment (matt-richardson) [inline docs/infrastructure/deployment-targets/dynamic-infrastructure/aws-accounts.md]: I presume these are fake?

File: docs/infrastructure/deployment-targets/dynamic-infrastructure/aws-accounts.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: Create AWS Account Command
+description: Cmdlet for creating an AWS account
+position: 10
```
Comment (matt-richardson) [inline docs/infrastructure/deployment-targets/dynamic-infrastructure/aws-accounts.md]: These all have the same position...

File: docs/infrastructure/deployment-targets/dynamic-infrastructure/aws-accounts.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+---
+title: Create AWS Account Command
+description: Cmdlet for creating an AWS account
```
Comment (matt-richardson) [inline docs/infrastructure/deployment-targets/dynamic-infrastructure/aws-accounts.md]: Consider
```suggestion
description: New-OctopusAwsAccount allows you to create an AWS account in Octopus from within a running deployment
```
and similar for the other ones

File: docs/infrastructure/deployment-targets/dynamic-infrastructure/aws-accounts.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+position: 10
+---
+
+## AWS Account
+Command: **_New-OctopusAwsAccount_**
+
```
Comment (matt-richardson) [inline docs/infrastructure/deployment-targets/dynamic-infrastructure/aws-accounts.md]: Feels like it needs a description here.

File: docs/infrastructure/deployment-targets/dynamic-infrastructure/aws-accounts.md (hunk 1 above)
Comment (hnrkndrssn) [reply docs/infrastructure/deployment-targets/dynamic-infrastructure/aws-accounts.md]: Yes, just a randomly generated string 🙂

File: docs/infrastructure/deployment-targets/dynamic-infrastructure/azure-accounts.md [hunk 5]
```diff
@@ -1,15 +1,17 @@
...
 
 ## Azure Service Principal Account
 Command: **_New-OctopusAzureServicePrincipalAccount_**
 
+_**New-OctopusAzureServicePrincipalAccount** allows you to create an Azure Service Principal account in Octopus from within a running deployment_
+
 | Parameters                    | Value                                                                                                      |
 |-------------------------------|------------------------------------------------------------------------------------------------------------|
-| `-name`                       | Name for the Octopus Service Principal account                                                             |
+| `-name`                       | Name for the Azure Service Principal account                                                             |
```
Comment (matt-richardson) [inline docs/infrastructure/deployment-targets/dynamic-infrastructure/azure-accounts.md]: Nice catch

File: docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md [hunk 6]
```diff
@@ -25,6 +25,9 @@ Each of the resource commands is available as a PowerShell function anywhere tha
 
 ### Accounts
 
+[Token Account](/docs/infrastructure/deployment-targets/dynamic-infrastructure/token-accounts.md)
```
Comment (matt-richardson) [inline docs/infrastructure/deployment-targets/dynamic-infrastructure/index.md]: Should this be alphabetical?

**Pull request merged**